extensions/functions_arithmetic.yaml

```diff
@@ -1352,6 +1352,51 @@ aggregate_functions:
         nullability: DECLARED_OUTPUT
         ordered: true
         return: LIST?<any>
+  
+  - name: "any_value"
+    description: >
+      Selects an arbitrary value from a group of values.
+
+      This function returns a single arbitrary value from an expression.
+      Main purpose of this function is to optimize a query containing a GROUP BY
+      caluse.
+
+      A comparison is not performed when this function is applied, hence it
+      is not necessary to be included in the GROUP BY clause.
+
+      This function returns NULL when expression produces NULL.
+      When the expression is NULL, it returns NULL for all values in the group.
```

Comment:
I'm not sure what's going on here. I would expect something like "If the input is empty, this returns null." and that's it, I don't see any special cases beyond that.